{"name": "test_create_and_drop_table", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'test_table' in <MagicMock name='mock.cursor.__enter__().fetchall()' id='139680627687824'>\n +  where <MagicMock name='mock.cursor.__enter__().fetchall()' id='139680627687824'> = get_tables('test_db')\n +    where get_tables = <test_db_adapter._MockAdapter object at 0x7f09eef72fe0>.get_tables", "trace": "connected_adapter = <test_db_adapter._MockAdapter object at 0x7f09eef72fe0>\n\n    def test_create_and_drop_table(connected_adapter: DbAdapter):\n        connected_adapter.create_db(_DB_NAME)\n        connected_adapter.create_table(_DB_NAME, _TABLE_NAME, _TABLE_DATA)\n>       assert _TABLE_NAME in connected_adapter.get_tables(_DB_NAME)\nE       AssertionError: assert 'test_table' in <MagicMock name='mock.cursor.__enter__().fetchall()' id='139680627687824'>\nE        +  where <MagicMock name='mock.cursor.__enter__().fetchall()' id='139680627687824'> = get_tables('test_db')\nE        +    where get_tables = <test_db_adapter._MockAdapter object at 0x7f09eef72fe0>.get_tables\n\ntests/test_db_adapter.py:54: AssertionError"}, "start": 1724682968427, "stop": 1724682968429, "uuid": "5788fc57-b37d-45c9-9c0a-30c22c848140", "historyId": "12c69485f69987dab214bbbbe859252a", "testCaseId": "12c69485f69987dab214bbbbe859252a", "fullName": "tests.test_db_adapter#test_create_and_drop_table", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_db_adapter"}, {"name": "host", "value": "krmb"}, {"name": "thread", "value": "52400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_db_adapter"}]}